src/Microsoft.DocAsCode.Build.Engine/HostService.cs

```diff
@@ -472,22 +472,33 @@ where changes.Contains(k)
             }
         }
 
-        public void SaveIntermediateModel(string cacheFolder)
+        public void SaveIntermediateModel(ModelManifest lmm, ModelManifest cmm)
         {
             var processor = Processor as ISupportIncrementalDocumentProcessor;
             if (processor == null)
             {
                 return;
             }
-            foreach (var f in Models)
+
+            foreach (var pair in _modelLoadInfo)
             {
-                string fileName = Path.Combine(cacheFolder, Path.GetFileName(f.File));
-                using (var stream = new FileStream(fileName, FileMode.OpenOrCreate, FileAccess.Write))
+                var fileName = Path.GetFileNameWithoutExtension(pair.Key.File);
+                if (pair.Value == LoadPhase.None)
                 {
-                    // processor.SaveIntermediateModel(f, stream);
-                    // copy files according to lastbuildinfo
-                    // update currentbuildinfo
+                    if (lmm == null)
+                    {
+                        throw new InvalidDataException($"Full build hasn't loaded model {pair.Key.FullPath}");
+                    }
+                    File.Copy(Path.Combine(lmm.BaseDir, fileName), Path.Combine(cmm.BaseDir, fileName));
```

Comment:
nice question~ that would be a problem

---
In reply to: [77568739](https://github.com/dotnet/docfx/pull/670#discussion_r77568739) [](ancestors = 77568739,77568436,77566882)